phoenix-core/src/main/java/org/apache/phoenix/util/SchemaUtil.java

```diff
@@ -1016,7 +1017,8 @@ public static TableName getPhysicalTableName(byte[] fullTableName, boolean isNam
     public static PName getPhysicalHBaseTableName(String tableName, boolean isNamespaceMapped, PTableType type) {
         if (!isNamespaceMapped) { return PNameFactory.newName(tableName); }
         return PNameFactory
-                .newName(tableName.replace(QueryConstants.NAME_SEPARATOR, QueryConstants.NAMESPACE_SEPARATOR));
+                .newName(StringUtils.replaceOnce(
```

Comment:
If we want to create the view NS."AAA.BBB", the original logic returns NS:AAA:BBB as a physical hbase table name. We need the name NS:AAA.BBB .